src/components/Main/Main.tsx

```diff
@@ -60,14 +59,18 @@ async function runSimulation(
     return
   }
 
-  if (params.population.cases !== "none" && !isRegion(params.population.cases)) {
```

Comment:
I see this fixed the `none` issue however it introduced type errors. I've refactored the code to allow us to handle this situation and to have everything type check!